curriculum/outlines/Linear DS.md

```diff
@@ -0,0 +1,39 @@
+# Linear DS
+---
+## Outline
+1. Static Array
+	 1. 1D Static Array
+	 2. 2D Static Array
+	 3. 3D Static Array
+2. Dynamically-Resizeable Array
+3. BitSet
+4. Bitmasks (Basic bit manipulation)
+	  * Example: Flipping i-th bit in an integer.
+5. LinkedLists
+6. Stacks
+7. Queues
+8. Dequeues
+    
```

Comment:
Add a section here for Material Links were each link is followed by the points or subpoint covered by this link. All outline points should be linked to an online tutorial, book section or supplementary tutorial as I mentioned above.